packages/storage/Readme.md

```diff
@@ -31,12 +31,14 @@ This is the simplest storage object in this toolkit. It is based on the similarl
 This object is meant to be initialized as a static constant in `state.rs`. However, it would also work perfectly fine if it was initialized during run time with a variable key (in this case though, you'd have to remind it what type of object is stored and its serde). Import it using the following lines:
 
 ```ignore
```

Comment:
Replace `ignore` with `rust`

Reply:
I think the ones that show the `use` lines have to stay ignored, because this code is inserted inside the `lib.rs` of each package, where this line does not work: `secret_toolkit::storage::...` but this does: `secret_toolkit_storage::...`. But if you import the package from the root of the secret toolkit then it would work just as in this example, I believe

Reply:
@toml01